flink-end-to-end-tests/test-scripts/test_kubernetes_embedded_job.sh

```diff
@@ -30,7 +30,7 @@ function cleanup {
     kubectl delete job flink-job-cluster
     kubectl delete service flink-job-cluster
     kubectl delete deployment flink-task-manager
-    rm -rf ${OUTPUT_VOLUME}
```

Comment:
Does `stop_kubernetes` delete the `${OUTPUT_VOLUME}`? Or why is this no longer needed?

Reply:
The `${OUTPUT_VOLUME}` is under `${TEST_DATA_DIR}`. And `${TEST_DATA_DIR}` will be cleaned up automatically when the e2e test finished. See `test-runner-common.sh#cleanup`.

I will add a hotfix to remove this line. Then it will be more clear.